criu/pipes.c

```diff
@@ -212,9 +212,9 @@ int restore_pipe_data(int img_type, int pfd, u32 id, struct pipe_data_rst **hash
 	munmap(pd->data, pd->pde->bytes);
 	pd->data = NULL;
 out:
-	ret = 0;
+	exit_code = 0;
 err:
-	return ret;
```

Comment:
Initially we had this err label to close image fd on error path ed456a1b3 ("pipe: restore pipe data"). Now when we don't have to do it can we remove this excess label and put proper "return 0/-1" everywhere?

Reply:
Fixed